packages/eslint-plugin/package.json

```diff
@@ -0,0 +1,71 @@
+{
+    "name": "@workleap/eslint-plugin",
+    "author": "Workleap",
+    "description": "Workleap ESLint rules and configs.",
+    "version": "0.0.0",
+    "keywords": [
+        "workleap",
+        "eslint",
+        "eslintconfig",
+        "eslintplugin",
+        "eslint-config",
+        "eslint-plugin"
+    ],
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/Workleap/wl-web-configs.git",
+        "directory": "packages/eslint-plugin"
+    },
+    "license": "Apache-2.0",
+    "main": "./dist/index.js",
+    "types": "./dist/index.d.ts",
+    "files": [
+        "docs",
+        "dist",
+        "CHANGELOG.md",
+        "README.md"
+    ],
+    "scripts": {
+        "build": "tsc",
+        "clean": "pnpm dlx rimraf dist node_modules",
+        "test": "jest",
```

Comment:
Does the test setup have to be per project?

It could be setuped only once at the root of the monorepo like it's done for Orbit

Reply:
In a bit monorepo like Orbit, we should split the test command in different projects. This would allow third party like TurboRepo and Nx to cache test results, and only run the tests on project that changed, instead of running on all the projects

Reply:
It also allows running at the root "pnpm test --filter @orbit-ui/components"